lib/SparqlEndpointFetcher.ts

```diff
@@ -38,6 +40,11 @@ export class SparqlEndpointFetcher {
     this.additionalUrlParams = args?.additionalUrlParams ?? new URLSearchParams();
     this.defaultHeaders = args?.defaultHeaders ?? new Headers();
     this.fetchCb = args?.fetch;
+    this.sparqlQueryParser = new SparqlParser({
```

Comment:
Can you also add a constructor option to pass a custom SPARQL parser?
We'll need it anyways for our future work, so let's already include it in this breaking change.